tests/framework/common.go

```diff
@@ -1239,7 +1240,13 @@ func (td *OsmTestData) Cleanup(ct CleanupType) {
 				},
 			)
 			if err != nil {
-				td.T.Logf("Poll err: %v", err)
+				td.T.Logf("Cleanup of namespaces timed out: %s", err)
+
+				// Dump the namespaces that failed to terminate
+				stdout, stderr, _ := td.RunLocal("kubectl", "get", "namespace", "-l", osmTest, "-o", "yaml")
+				td.T.Logf("Namespace info:")
+				td.T.Logf("stdout:\n%s", stdout)
+				td.T.Logf("stderr:\n%s", stderr)
```

Comment:
For something this simple, could you use the k8s Go client instead to do this? There are only a couple places we invoke `kubectl` directly and I'd rather not add more if we can help it.

Reply:
I'd rather not, in the interest of simplifying how the object is dumped nicely by kubectl.